### PR TITLE
CC Article for SSI guides

### LINKED ISSUE #675: Mine SSI guides for curated content
author: @pagrubel 
EB member: @bartlettroscoe 

The UK Software Sustainability Institute has a collection of "guides" they've produced over the years.  Think of them as roughly equivalent to our original articles.

I think it would be a good idea for BSSw to point to many/all of them.  Not sure how to package them (one cc, multiple ccs for groups, or individual ccs).

https://www.software.ac.uk/resources/guides

### PR DESCRIPTION
**Pull request by pagrubel**
EB Member: @bartlettroscoe 

* Add CC article for SSI guides
* Article pointing to the website for the Software Sustainability Institute Guides
Resolves #675

## PR checklist for files displayed on bssw.io site

* [x] ***[Author]*** `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
* [x] ***[Author]*** Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
* [x] ***[Author]*** Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
* [x] ***[EB Mem]*** Assign this PR to the EB member `<eb-member-id>`.
* [x] ***[EB Mem]*** Assign this PR to the author of the PR `<pr-author-id>`.
* [x] ***[EB Mem]*** Add one or more Reviewers.
* [x] ***[EB Mem]*** Add label `content: <content-type>` for the type of contribution.
* [x] ***[EB Mem]*** Add to Project `Content Development` (see [Content Development]).
* [x] ***[EB Mem]*** Add [meta-data] to the `*.md` file(s) (set `Publish: preview`).
* [x] ***[EB Mem]*** Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
* [x] ***[EB Mem]*** Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
* [x] ***[Author]*** Make any final changes to the PR based on feedback.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
* [x] ***[EB Mem]*** Rebuild [preview] site and re-confirm content looks correct.
* [x] ***[EB Mem]*** Ensure at least one reviewer signs off on the final changes.
* [x] ***[EB Mem]*** Change meta-data to `Publish: yes` and commit if fully ready to publish.
* [x] ***[EB Mem]*** Move the PR to "Ready to Publish" in [Content Development].
* [x] ***[EB Mem]*** Leave comment and @mention `@betterscientificsoftware/bssw-maint` (**BSSw Maint**) asking to carry out final publication steps.
* [x] ***[BSSw Maint]*** Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
* [x] ***[BSSw Maint]*** Merge PR. (Should automatically move to "Done" in [Content Development].)
* [ ] ***[BSSw Maint]*** Verify new contribution shows up on [bssw.io] as expected.

NOTE:
* Checklist items prefixed with ***[Author]*** are expected to be performed by the author of the PR or can be performed by the author.
* Checklist items prefixed with ***[EB Mem]*** must be performed by a [BSSw.io Editorial Board (EB) Member](https://betterscientificsoftware.github.io/bssw.io/bssw_members.html).
* Checklist items prefixed with ***[BSSw Maint]*** must be performed by someone in the `@betterscientificsoftware/bssw-maint` team (hint: type `@`, `b`, `s`, `s`, `w`, `-`, `m`  to auto-complete to `@betterscientificsoftware/bssw-maint`).

### CODE REVIEW

File: CuratedContent/SoftwareSustainabilityInstituteGuides.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+The Software Sustainability Institute (SSI), based in the UK,
+promotes the importance of scientific research software. The SSI
+website presents over 80 guides to aid your team to protect
+software into the future. The guides are categorized by the
+following research software project roles: researchers,
+developers, managers and instructors or material developers. The
+categories and descriptions make it easy to navigate to guides
+that are valuable to each of your team members. 
+
+Managers, researchers and developers can peruse a variety of topics such as building community, recruiting champions and students, managing research projects, data management plans and approaches to sustainability. Developers can take a deeper dive on subjects like making software reusable, publicizing software, best practices, and the challenges of dependencies. While researchers will find topics on publishing and the importance of openness in research helpful. The site also hosts guides for training and training materials including specific help for organizing and teaching Carpentry workshops.
```
Comment (bartlettroscoe) [inline CuratedContent/SoftwareSustainabilityInstituteGuides.md:25]: > While researchers will find topics on publishing and the importance of openness in research helpful.

@pagrubel, I don't think that is a complete sentence.  I think you can make this a sentence by removing the word "While".

**Review by bartlettroscoe — CHANGES_REQUESTED** [review]
@pagrubel, the article is nice and short and reads well.  I found only one issue that needs to be fixed before it is published.  I will post a stacked PR against this branch with a suggestion to fix.

File: CuratedContent/SoftwareSustainabilityInstituteGuides.md (hunk 1 above)
Comment (bartlettroscoe) [reply CuratedContent/SoftwareSustainabilityInstituteGuides.md:25]: @pagrubel, the fix is in the [stacked PR](https://bssw.io/items/pull-request-size-matters#sfer-ezikiw-7) #1075.  That PR just needs to be accepted and merged to this PR branch and then this PR can be approved and merged.

**Comment by pagrubel** [comment]
@bartlettroscoe Merged your fix, thanks.

**Comment by pagrubel** [comment]
@bartlettroscoe I added a focus, let me know what you think

**Comment by pagrubel** [comment]
@bartlettroscoe just waiting for your approval

**Review by bartlettroscoe — APPROVED** [review]
Looks good.  Thanks!

**Comment by pagrubel** [comment]
@betterscientificsoftware/bssw-maint This PR is ready for final publication procedure.

**Review by rinkug — APPROVED** [review]

**Comment by rinkug** [comment]
Published: https://bssw.io/items/software-sustainability-institute-guides

**Pull request merged**